schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
  date_of_birth DATE,
  escape_attempts INT,
  neutered BOOLEAN,
- weight_kg NUMERIC
+ weight_kg NUMERIC,
+ species TEXT
```

Comment:
- [ ] @samuelmoiwa great work on implementing the changes requested, however, you didn't use the best practice to update the schema, you need to use the Alter keyword eg ` ALTER TABLE table-name ADD species varchar(255);`, hope the requirements are now clear, kindly contact me on slack if need of more help.
![image](https://user-images.githubusercontent.com/61408860/207528294-2cfab02f-6285-4674-81e7-e2f9d9b26a3a.png)